projects/drivers/xilinx_platform/platform_drivers.c

```diff
@@ -180,55 +195,64 @@ int32_t spi_init(struct spi_desc **desc,
 		 const struct spi_init_param *param)
 {
 	spi_desc *descriptor;
+	xil_spi_desc *xil_descriptor;
+	xil_spi_init_param *xil_param;
 	int32_t ret;
 
-	descriptor = (struct spi_desc *) calloc(1, sizeof(*descriptor));
+	descriptor = calloc(1, sizeof(*descriptor));
```

Comment:
nitpick: no need for parenthesis in `sizeof(*descriptor)`; it can be just `sizeof *descriptor`.
This can be checked in other places as well. `sizeof` only requires parenthesis when it is done with a type (e.g. `sizeof (int)`), or with an expression (e.g. `sizeof (a+b)`).